src/mdlink.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+const fs = require('fs');
+const marked = require('marked');
```

Comment:
Asi se usa const esto esta genial
```suggestion
const path = require('path');
const fs = require('fs');
const marked = require('marked');
```